### ISSUE TITLE
TF_binding_site child terms

### ISSUE THREAD

**Issue by RLovering** [The-Sequence-Ontology/SO-Ontologies#466]
Rather than giving the dbTF binding site terms a specific regulatory parent (like enhancer) I woudl like to suggest that SO has a structure along these lines:

- transcriptional_cis_regulatory_region

> Promoter (core promoter)
>> new term eukaryotic promoter
>>> various RNA polymerase specific promoter child terms
>> new term prokaryotic promoter
>> various other promoters - inducible, bidirectional ....
> promoter_flanking_region  
> enhancer

- transcription_regulatory_region

> TF_binding_site
>> pheromone_response_element
>> ISRE  
>> androgen_response_element   
>> retinoic_acid_responsive_element

**Comment by davidwsant** [comment]
Hi Ruth, I am the new postdoc in Karen's lab and I will be helping update the sequence ontology. My background is in human genetics and transcription, so I am now trying to look back at the new issues that have been posted. 
I agree that RARE should be moved to TF_binding_site. I saw that in one of your earlier posts and I still plan to update that. 
It looks like you are suggesting that we split core_promoter_element into two separate pieces, one for prokaryotes and one for eukaryotes. Is that correct?

**Comment by RLovering** [comment]
Hi David
sorry I should have got back to your last email. To be honest I am hoping to get an agreement from the GREEKC experts about this when they next meet. They have come up with a good new definition for promoter, prokaryote promoter and eukaryote promoter, but I actually think this is probably a good definition for core promoter, prokaryote core promoter and eukaryote core promoter.

in addition, I don't know anything about RNA pol I and III promoters, but I have a feeling that probably the definition describes only the RNA pol II core promoter. 

However, I would like the experts to look at this and make any additional comments before you go ahead and do this. Unfortunately I will not be at the next meeting but Pascale has agreed to present these options at the meeting, which is next week. 

What do you think about this?

Ruth

**Comment by davidwsant** [comment]
Hi Ruth,

Thanks for the quick response. I have created a spreadsheet that I have passed to a couple of others working on SO to ask for their help. I have about 5 or 6 other terms you have commented on that also need to be updated. Of particular note for promoter related terms, core_promoter_element currently has no definition. Let me know what they decide about splitting the core promoter element and I will run it by the group and try to update it as soon as I can. 

Thanks,

Dave
